### ISSUE TITLE
navigate to known hosts edit page on host public key mismatch notification

### ISSUE THREAD

**Issue by kcking** [kryptco/kryptonite-ios#62]

**Comment by loeken** [comment]
yes please, the known hosts location on the developers tab is hard to spot, only found the info from another issue, which linked to here.

**Comment by loeken** [comment]
also a search field there would be useful, kinda takes a bit time to find it out of the few hundred hosts i have added :)

**Comment by Esity** [comment]
I don't like to typically comment on issues but this one still seems important. Do we think this will get any traction?
